app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources
+        xmlns:tools="http://schemas.android.com/tools"
+        tools:ignore="MissingTranslation">
+        <string name="app_name">Kaffee</string>
+        <string name="keep_display_on">Bildschirm eingeschaltet lassen</string>
+        <string name="tap_to_turn_off">Tippen zum Ausschalten</string>
+        <string name="hint_1">Diese App stellt in den Schnelleinstellungen (Das Menü, wo man W-LAN und Bluetooth ein- und ausschalten kann) eine Kachel zur Verfügung, welche es erlaubt, den Bildschirm eingeschaltet zu lassen, ohne die Geräteeinstellungen zu ändern</string>
+        <string name="hint_2">Kaffee kann durch Klicken auf den Titel oder die Benachrichtigung oder das manuelle Ausschalten des Bildschirms gestoppt werden</string>
```

Comment:
```suggestion
        <string name="hint_2">Kaffee kann durch Tippen auf die Kachel oder die Benachrichtigung oder durch das manuelle Ausschalten des Bildschirms gestoppt werden</string>
```

Reply:
I must have misread the english text. Thanks for correcting